### ISSUE TITLE
Web accessibility - Very Low Contrast

### ISSUE THREAD

**Issue by natnael2012** [learningequality/ka-lite#3297]
Branch - Develop 

Expected Behavior - Text should be present by having a contrast ratio greater than 4.5:1, or large texts (larger than 18 point or 14 point bold) should have a contrast ratio greater than 3:1. 
Standards and Guidelines [ 1.4.3 Contrast (Minimum) (Level AA) http://webaim.org/standards/wcag/checklist#sc1.4.3 ]

Current Behavior - The menu Items has Contrast Ratio: 2.91:1, (background color #5aa685 and foreground color #fff). This is very low contrast between foreground and background colors. 

Elements with background images must have a background color defined that provides adequate contrast when images are disabled or unavailable. Adequate contrast is necessary for all users, especially users with low vision.

Snapshot from the develop branch being inspected by Wave checker
![contrast](https://cloud.githubusercontent.com/assets/8178368/6650818/a47df280-c9e2-11e4-8f36-4364856f17bb.png)

**Comment by MCGallaspy** [comment]
It's not clear to me how ambitious this is for 0.14 -- can someone (@jtamiace maybe) weigh in on the feasibility of resolving this before the version release? We can change the milestone if necessary.

**Comment by radinamatic** [comment]
The feature proposal for this is in #3921.

**Comment by aronasorman** [comment]
Superseded by #3921.
